### ISSUE TITLE
data xP3 not suitable for chatbot tranining or finetuning?

### ISSUE THREAD

**Issue by yzxyzh** [PhoebusSi/Alpaca-CoT#75]
Hi:

I find that the dataset xP3 contains much more data than all the others combine. But it's more like a dataset that can be used to train a basemodel, rather than a dataset prepared for a chatbot. Do anyone have any idea how or where I can find the transformed dataset that shall turn the xP3 into a chat-based dataset?

Thanks.

**Comment by yhyu13** [comment]
Xp3 is big due to multilingual

**Comment by PhoebusSi** [comment]
Yes, xp3 may not be suitable as a chat based dataset, as it consists of many multilingual nlp tasks. If your goal is to test on an existing nlp dataset, you may need some data from xP3, otherwise it is recommended to use other instruction data.
